src/v3/src/components/Checkbox/Checkbox.tsx

```diff
@@ -75,7 +84,14 @@ const Checkbox: UISchemaElementComponent<UISchemaElementComponentWithValidationP
         )}
         label={(
           <Fragment>
-            {label}
+            {labelInfo?.noTranslate ? (
+              <Box
+                component="span"
+                className="no-translate"
```

Comment:
Is there any undesirable behavior if this `Box` is always present and only add the `className` conditionally like the other places?

Reply:
This scenario should never happen, but if for some reason someone configured this incorrectly that caused this to happen, it would not have a negative impact on the UI as it would just display an empty span with that classname, and the class doesnt do anything other than allow our selenium tests to avoid asserting on it for translated (ok-pl) text.

Reply:
Oh I meant if we could restructure this like this:
```tsx
<Fragment>
  <Box
    component="span"
    className={labelInfo?.noTranslate ? 'no-translate' : undefined}
  >
    {labelInfo?.value as string}
  </Box>
//...
```

Reply:
Ohhh, ok actually, I did originally consider this, the reason I went back on it was because it caused alot of (unnecessary) noise in the snapshot files. So to limit the number of changes in snaopshots I did this. But it would not hurt. Did you want me to do it this way?

Reply:
Oh I see, I don't feel strongly, so up to you whichever you prefer